openvpn/mbedtls/crypto/cipheraead.hpp

```diff
@@ -19,7 +19,7 @@
 //    along with this program in the COPYING file.
 //    If not, see <http://www.gnu.org/licenses/>.
 
-// Wrap the mbed TLS AEAD API.
+// Wrap the mbed TLS AEAD API. Based on ciphergcm.hpp by ProMIND
```

Comment:
I don't think this comment should be there. Also this sound like it is based on someone elses work. Do you have the full permission/right in terms of the CLA?

Reply:
Hi there! I have used the tag "ProMIND" in the modified/added parts in order to let the maintainers to easily spot all the changes I have made. You can delete them, of course.
They are simply meant to tag the modifications or additions.
The implementation of ChaCha20 has been made by me and it is not based on any other work.